plugins/modules/openbsd_pkg.py

```diff
@@ -152,7 +152,11 @@ def execute_command(cmd, module):
     # This makes run_command() use shell=False which we need to not cause shell
     # expansion of special characters like '*'.
     cmd_args = shlex.split(cmd)
-    return module.run_command(cmd_args)
+
+    # We set TERM to 'dumb' to keep pkg_add happy if the machine running
+    # ansible is using a TERM that the managed machine does not know about,
```

Comment:
I don't see how `TERM` set on the Ansible controller has any influence what is run on the remote machine. Both this comment and the formulation in the changelog fragment thus look wrong to me. Could it be that the remote machine is setting `TERM=xterm-kitty` somewhere itself?

Reply:
In your example: `$ TERM=xterm-kitty ansible -i hosts openbsd_pkg_tests -u root -m community.general.openbsd_pkg  -a 'name=jq state=present'`

Is `openbsd_pkg_tests` something that ends up as `localhost`? Also what happens if you change the start to `TERM=somerandomstring`, does that string also appear in the output then?

Reply:
No `localhost` involved. The `openbsd_pkg-tests` target only includes a publicly routable IP address. Ansible was running in a ubuntu 22.04 VM on my local machine, and the OpenBSD host targeted by `openbsd_pkg-tests` was a VM running at a public VPS provider (as you can see in the output).

Setting something else fails as well (in a state before the patch was applied):
```
$ TERM=randomstring ansible -i hosts openbsd_pkg_tests -u root -m community.general.openbsd_pkg  -a 'name=jq state=present'
[WARNING]: You are running the development version of Ansible. You should only
run Ansible from "devel" if you are modifying the Ansible engine, or trying out
features under development. This is a rapidly changing source of code and can
become unstable at any point.
[WARNING]: packaging Python module unavailable; unable to validate collection
Ansible version requirements
78.141.209.142 | FAILED! => {
    "build": false,
    "changed": false,
    "msg": "No progress meter: failed termcap lookup on randomstring\n",
    "name": [
        "jq"
    ],
    "state": "present"
}
```

I am pretty sure this is a result of `ssh` sending over the `TERM` setting, from `ssh_config(5)` on my 22.04:
```
     SendEnv
             Specifies what variables from the local environ(7) should be sent
             to the server.  The server must also support it, and the server
             must be configured to accept these environment variables.  Note
             that the TERM environment variable is always sent whenever a
             pseudo-terminal is requested as it is required by the protocol.
```

Reply:
Aaah, yes, it will be SSH passing it on. Thanks!